src/main/app/src/app/informatie-objecten/informatie-object-view/informatie-object-view.component.html

```diff
@@ -202,6 +202,23 @@
                       ></read-more>
                     </td>
                   </ng-container>
+                  <ng-container matColumnDef="actie">
+                    <th mat-header-cell *matHeaderCellDef>
+                      {{ "actie" | translate }}
+                    </th>
+                    <td mat-cell *matCellDef="let regel">
+                      <read-more
+                        [text]="
+                          regel.actie
+                            ? ('action.document.' + regel.actie
+                              | translate
+                              | empty)
+                            : 'TBD'
```

Comment:
With https://github.com/infonl/dimpact-zaakafhandelcomponent/pull/1456 `regel.actie` can still be null if unknown action is fetched from OpenZaak.

Perhaps this should read something like "unknown":
```suggestion
                            : 'onbekend'
```

Reply:
OK; good one; fixed ✅